<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="Export">
        <attribute name="title">
         <string>Export</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="path_label">
               <property name="text">
                <string>Path Export : </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="path_export">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="font">
                <font>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="browseX_pushButton">
               <property name="text">
                <string>browse</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="export_tableWidget">
             <column>
              <property name="text">
               <string>Export</string>
              </property>
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Object</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Variants</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Output</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="add_button">
               <property name="text">
                <string>Add</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="remove_button">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Remove</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="export_all_checkBox">
             <property name="text">
              <string>Export All Selection</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="set_xform_checkBox">
             <property name="text">
              <string>Set Xform</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="export_button">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>100</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="sizeIncrement">
              <size>
               <width>0</width>
               <height>100</height>
              </size>
             </property>
             <property name="text">
              <string>Export</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="Import">
        <attribute name="title">
         <string>Import</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="newusd_label">
             <property name="text">
              <string>Path Import :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="path_import"/>
           </item>
           <item>
            <widget class="QPushButton" name="browse_pushButton">
             <property name="text">
              <string>browse</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="import_listWidget"/>
         </item>
         <item>
          <widget class="QPushButton" name="import_pushButton">
           <property name="text">
            <string>Import</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
